todo-list-ui/src/app/todo-actions/todo-actions.component.html

```diff
@@ -0,0 +1,14 @@
+<app-todo-edit-modal
+  [id]="id"
+  [title]="title"
+  (newTitle)="$event ? editTodo($event) : undefined"
+></app-todo-edit-modal>
+<app-confirmation-modal
+  [message]="'Are you sure to delete this todo?'"
```

Comment:
This is worded wrong, maybe you meant `Are you sure you want to delete this todo?` or maybe simpler `Delete todo?`